snow/consensus/snowman/topological.go

```diff
@@ -291,7 +291,7 @@ func (ts *Topological) RecordPoll(ctx context.Context, voteBag bag.Bag[ids.ID])
 	ts.pollNumber++
 
 	var voteStack []votes
-	if voteBag.Len() >= ts.params.Alpha {
+	if voteBag.Len() >= ts.params.AlphaPreference {
```

Comment:
q: here we pick the smallest of the alphas, to be able to update preferences or polls (or both), right?